motionplan/constraint.go

```diff
@@ -94,6 +95,14 @@ func (c *constraintHandler) AddConstraint(name string, cons Constraint) {
 	}
 }
 
+// AddConstraint will add or overwrite constraint functions with the ones present in the specified other constraintHandler.
+// A constraint function should return true if the given position satisfies the constraint.
+func (c *constraintHandler) AddConstraints(other *constraintHandler) {
```

Comment:
Not a huge fan of the name; maybe "merge constraint handlers" or something? "AddConstraints" makes me expect to pass in an array or something of that sort.

Reply:
`MergeConstraintHandlers` is good with me